bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx

```diff
@@ -42,6 +47,18 @@ class NavBar extends Component {
         </div>
         <div className={styles.center}>
           <h1 className={styles.presentationTitle}>{presentationTitle}</h1>
+          <span className={styles.divideBar}> | </span>
+          {beingRecorded == 'true' ?
+            <svg width="30" height="30" className={styles.recordImage}>
+              <circle cx="13" cy="17" r="10" stroke="#ffffff" strokeWidth="1" fill="#2A2D36" />
+              <circle cx="13" cy="17.5" r="5" stroke="#2A2D36" strokeWidth="1" fill="#F0615F" />
+            </svg>
+            :
+            <svg width="30" height="30" className={styles.recordImage}>
+              <circle cx="13" cy="17" r="10" stroke="#ffffff" strokeWidth="1" fill="#2A2D36" />
+              <circle cx="13" cy="17.5" r="5" stroke="#2A2D36" strokeWidth="1" fill="#ffffff" />
+            </svg>
```

Comment:
You could avoid the code duplication by storring the fill value into a variable and change the variable value if beingRecorded is true.